gdraw/gmenu.c

```diff
@@ -1907,23 +1907,23 @@ int GMenuBarCheckKey(GWindow top, GGadget *g, GEvent *event) {
     }
 #endif
 
-    TRACE("about to look for hotkey in new system...state:%d keysym:%d\n", event->u.chr.state, event->u.chr.keysym );
-    TRACE("     has ksm_control:%d\n", (event->u.chr.state & ksm_control ));
-    TRACE("     has ksm_meta:%d\n",    (event->u.chr.state & ksm_meta ));
-    TRACE("     has ksm_shift:%d\n",   (event->u.chr.state & ksm_shift ));
+//    TRACE("about to look for hotkey in new system...state:%d keysym:%d\n", event->u.chr.state, event->u.chr.keysym );
```

Comment:
Why are we disabling these traces? Were they producing annoying output somewhere?